From 8a559e2ff9aa875656c3c62df16dedc84f68c2b7 Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Wed, 16 Jun 2021 20:38:07 +0200
Subject: [PATCH 11/96] ARM: dts: meson: Add GPIO controller capabilities to
 the first USB PHY

This is needed for boards that implement OTG functionality to read out
the value of the "ID" signal (e.g. on Micro USB connectors).

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
---
 arch/arm/boot/dts/amlogic/meson.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/amlogic/meson.dtsi b/arch/arm/boot/dts/amlogic/meson.dtsi
index 8cb0fc78b..0e7756c95 100644
--- a/arch/arm/boot/dts/amlogic/meson.dtsi
+++ b/arch/arm/boot/dts/amlogic/meson.dtsi
@@ -124,6 +124,8 @@ usb0_phy: phy@8800 {
 				compatible = "amlogic,meson-mx-usb2-phy";
 				#phy-cells = <0>;
 				reg = <0x8800 0x20>;
+				gpio-controller;
+				#gpio-cells = <2>;
 				status = "disabled";
 			};
 
-- 
2.45.1

